SIL.ReleaseTasks/SetReleaseNotesProperty.cs

```diff
@@ -25,16 +25,26 @@ public class SetReleaseNotesProperty : Task
 
 		public string AppendToReleaseNotesProperty { get; set; }
 
+		public Boolean FilterEntries { get; set; }
```

Comment:
```suggestion
		public bool FilterEntries { get; set; }
```

By convention the language keywords instead of the BCL types are prefered (cf https://github.com/dotnet/runtime/issues/13976)